toolkit/tools/pkgworker/pkgworker.go

```diff
@@ -385,6 +386,14 @@ func copyFilesIntoChroot(chroot *safechroot.Chroot, srpmFile, repoFile, rpmmacro
 		filesToCopy = append(filesToCopy, rpmmacrosCopy)
 	}
 
+	if runCheck {
+		resolvFileCopy := safechroot.FileToCopy{
```

Comment:
Lets print a warning here, something like `logger.Log.Warn("Enabling networking for check section")`.